hack/windows/Install-OVS.ps1

```diff
@@ -259,8 +264,10 @@ DownloadOVS
 
 InstallOVS
```

Comment:
Actually, what we need is to only install the OVS kernel driver. The OVS container should be able to run without any OpenvSwitch path created. I'm not sure if the current image can support running the container without the "C:\openvswitch" directory on the host. You can decide whether to change it in 1.13 based on the current bandwidth (assuming it will involve script, yaml and snapshot changes). Our final goal is to run the OVS container on a clean host environment with only the kernel driver installed. @rajnkamr @NamanAg30 @Atish-iaf 

cc @wenyingd

Reply:
@XinShuYang ,Currently kernel driver is installed on host and directories like c:/openvswitch/etc/openvswitch" and "c:/openvswitch/var/run/openvswitch" are created by Install-OVS.ps1 , 
Current image is not having these directories hence it needs to exist on host as conf.db file needs to persistent.

Reply:
[5249](https://github.com/antrea-io/antrea/issues/5249)